components/feature/session/src/main/java/mozilla/components/feature/session/FullScreenFeature.kt

```diff
@@ -28,6 +30,11 @@ open class FullScreenFeature(
     }
 
     override fun onFullScreenChanged(session: Session, enabled: Boolean) = fullScreenChanged(enabled)
+    override fun onMetaViewportFitChanged(session: Session, layoutInDisplayCutoutMode: Int) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
```

Comment:
Do we need this check here if we already have it in the engine?

Reply:
you're right.  Forgot to clean it up.  Thanks,